functests/1_performance/performance.go

```diff
@@ -222,7 +220,7 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 		It("[test_id:42400][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running as sched_fifo", func() {
 			for _, node := range workerRTNodes {
-				pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "stalld"}, &node)
+				pid, err := nodes.ExecCommandOnNode([]string{"pidof", "stalld"}, &node)
 				Expect(err).ToNot(HaveOccurred())
```

Comment:
@fromanirh : 
what's the advantage of using pidof?

 
@shajmakh :
using "pidof" we get the pid of running program called "stalld", while when using "pgrep" it will list us all the running processes that their name matches the name after pgrep.Thus I found it more reasonable to replace "pgrep" with "pidof" when searching for stalld's pid ,since stalld is only one process running.